lib/mhv_logging/configuration.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+require 'rx/configuration'
+module MHVLogging
+  # Configuration class used to setup the environment used by client
+  class Configuration < Rx::Configuration
```

Comment:
I'll change this then and possibly look at naming the configuration something like MHVConfiguation vs Rx::Configuration.